batch_seqr_loader/batch_workflow.py

```diff
@@ -0,0 +1,567 @@
+@click.option(
+    '--vep-config', 'vep_config_json_path', help='Path of hail vep config .json file'
+)
+@click.option('--vep-block-size', 'vep_block_size')
+def main(
+    gvcf_buckets: List[str],
+    genomicsdb_bucket: str,
+    ped_fpath: str,
+    dataset_name: str,
+    dest_mt_path: str,
+    work_bucket: str,
+    keep_scratch: bool,
+    reuse_scratch_run_id: Optional[str],
+    dry_run: bool,
+    billing_project: Optional[str],
+    genome_version: str,  # pylint: disable=unused-argument
+    disable_validation: bool,  # pylint: disable=unused-argument
+    dataset_type: str,  # pylint: disable=unused-argument
+    sample_type: str,  # pylint: disable=unused-argument
+    remap_path: str = None,  # pylint: disable=unused-argument
+    subset_path: str = None,  # pylint: disable=unused-argument
+    vep_config_json_path: Optional[str] = None,  # pylint: disable=unused-argument
+    vep_block_size: Optional[int] = None,  # pylint: disable=unused-argument
+):
+    """
+    Entry point for a batch workflow
+    """
+    billing_project = os.getenv('HAIL_BILLING_PROJECT') or 'seqr'
+
+    hail_bucket = os.environ.get('HAIL_BUCKET')
+    if not hail_bucket or keep_scratch or reuse_scratch_run_id:
+        # Scratch files are large, so we want to use the temporary bucket for them
+        hail_bucket = f'{work_bucket}/hail'
+    logger.info(
+        f'Starting hail Batch with the project {billing_project}, '
+        f'bucket {hail_bucket}'
+    )
+    backend = hb.ServiceBackend(
+        billing_project=billing_project,
+        bucket=hail_bucket.replace('gs://', ''),
+    )
+    b = hb.Batch('Seqr loader', backend=backend)
+
+    samples_df = find_inputs(gvcf_buckets, ped_fpath=ped_fpath)
+
+    reference = b.read_input_group(
+        base=REF_FASTA,
+        fai=REF_FASTA + '.fai',
+        dict=REF_FASTA.replace('.fasta', '').replace('.fna', '').replace('.fa', '')
+        + '.dict',
+    )
+    dbsnp = b.read_input_group(base=DBSNP_VCF, idx=DBSNP_VCF + '.idx')
+
+    gathered_vcf_path = join(work_bucket, f'{dataset_name}.vcf.gz')
+    if not file_exists(gathered_vcf_path):
+        gather_job = _make_jobs_that_produce_vcf(
+            b=b,
+            genomicsdb_bucket=genomicsdb_bucket,
+            samples_df=samples_df,
+            output_vcf_path=gathered_vcf_path,
+            reference=reference,
+            dbsnp=dbsnp,
+            hail_bucket=hail_bucket,
+            work_bucket=work_bucket,
+            reuse_scratch_run_id=reuse_scratch_run_id,
+        )
+    else:
+        gather_job = b.new_job('Joint-call VCF')
+
+    dataproc.hail_dataproc_job(
+        b,
+        f'PYTHONPATH=$(pwd):$(pwd)/luigi_pipeline:$PYTHONPATH '
+        f'batch_seqr_loader/seqr_load.py '
+        f'--source-path {gathered_vcf_path} '
+        f'--dest-mt-path {dest_mt_path} '
+        f'--bucket {join(work_bucket, "seqr_load")} ',
+        max_age='8h',
+        packages=DATAPROC_PACKAGES,
+        pyfiles=['luigi_pipeline/lib', ''],
```

Comment:
```suggestion
        pyfiles=['luigi_pipeline/lib', 'hail_scripts'],
```
- hail_scripts is imported here: https://github.com/populationgenomics/hail-elasticsearch-pipelines/blob/856638079a8d150a41ac6e357894c704461a40e9/batch_seqr_loader/seqr_load.py#L15
- Do these paths work relative to the repo base? 
- Might need to update the python path with hail_scripts?

Reply:
Right, forgot about `hail_scripts`! Thanks :)